test/screenshot/resemble.json

```diff
@@ -1,5 +1,5 @@
 {
   "output": {
-    "transparency": 0.2
+    "transparency": 0
```

Comment:
Does this make it fully transparent of fully opaque?

Reply:
Fully transparent. It only outputs the pink "diff" pixels now, per Andrey's suggestion.